APUOPE-RE/frontend
# Search bar and "export as pdf" button

**How to review:**

1. Make sure there is a working search bar for previous quizzes 
![image](https://github.com/user-attachments/assets/56b585f3-0a78-42d6-abd7-28a6886f16ae)

2. Make sure there's an "export as pdf" button below the quiz results
![image](https://github.com/user-attachments/assets/22c1289b-cb52-4677-a7a9-aa37ce2ec7ce)
